mne/decoding/time_gen.py

```diff
@@ -914,18 +544,681 @@ def _predict(X, estimators):
+                               tlim=tlim, ax=ax, cmap=cmap, show=show,
+                               colorbar=colorbar, xlabel=xlabel, ylabel=ylabel)
+
+    def plot_diagonal(self, title=None, xmin=None, xmax=None, ymin=None,
+                      ymax=None, ax=None, show=True, color=None,
+                      xlabel=True, ylabel=True, legend=True, chance=True,
+                      label='Classif. score'):
+        """Plotting function of GeneralizationAcrossTime object
+
+        Plot each classifier score trained and tested at identical time
+        windows.
+
+        Parameters
+        ----------
+        title : str | None
+            Figure title.
+        xmin : float | None, optional
+            Min time value.
+        xmax : float | None, optional
+            Max time value.
+        ymin : float | None, optional
+            Min score value. If None, sets to min(scores).
+        ymax : float | None, optional
+            Max score value. If None, sets to max(scores).
+        ax : object | None
+            Instance of mataplotlib.axes.Axis. If None, generate new figure.
+        show : bool
+            If True, the figure will be shown. Defaults to True.
+        color : str
+            Score line color.
+        xlabel : bool
+            If True, the xlabel is displayed. Defaults to True.
+        ylabel : bool
+            If True, the ylabel is displayed. Defaults to True.
+        legend : bool
+            If True, a legend is displayed. Defaults to True.
+        chance : bool | float. Defaults to None
+            Plot chance level. If True, chance level is estimated from the type
+            of scorer.
+        label : str
+            Score label used in the legend. Defaults to 'Classif. score'.
+
+        Returns
+        -------
+        fig : instance of matplotlib.figure.Figure
+            The figure.
+        """
+        return plot_gat_times(self, train_time='diagonal', title=title,
+                              xmin=xmin, xmax=xmax,
+                              ymin=ymin, ymax=ymax, ax=ax, show=show,
+                              color=color, xlabel=xlabel, ylabel=ylabel,
+                              legend=legend, chance=chance, label=label)
+
+    def plot_times(self, train_time, title=None, xmin=None, xmax=None,
+                   ymin=None, ymax=None, ax=None, show=True, color=None,
+                   xlabel=True, ylabel=True, legend=True, chance=True,
+                   label='Classif. score'):
+        """Plotting function of GeneralizationAcrossTime object
+
+        Plot the scores of the classifier trained at specific training time(s).
+
+        Parameters
+        ----------
+        train_time : float | list or array of float
+            Plots scores of the classifier trained at train_time.
+        title : str | None
+            Figure title.
+        xmin : float | None, optional
+            Min time value.
+        xmax : float | None, optional
```

Comment:
IIRC, I think I got that from @Eric89GXL. @Eric89GXL, do you have an opinion on this?